src/main/fc/init.c

```diff
@@ -95,14 +95,15 @@
 #include "fc/tasks.h"
 
 #include "flight/alt_hold.h"
+#include "flight/autopilot.h"
 #include "flight/failsafe.h"
 #include "flight/imu.h"
 #include "flight/mixer.h"
 #include "flight/gps_rescue.h"
 #include "flight/pid.h"
 #include "flight/pid_init.h"
 #include "flight/position.h"
-#include "flight/position_control.h"
+#include "flight/pos_hold.h"
```

Comment:
???

Reply:
`position_control.h` is now re-named to `autopilot.h`
sorry, shouldn't have added pos_hold.h, brain fade from working on position hold at the same time as this PR.
thanks